package-lock.json

```diff
@@ -2777,18 +2790,17 @@
       }
     },
     "node_modules/@puppeteer/browsers": {
-      "version": "2.7.1",
-      "resolved": "https://registry.npmjs.org/@puppeteer/browsers/-/browsers-2.7.1.tgz",
-      "integrity": "sha512-MK7rtm8JjaxPN7Mf1JdZIZKPD2Z+W7osvrC1vjpvfOX1K0awDIHYbNi89f7eotp7eMUn2shWnt03HwVbriXtKQ==",
+      "version": "2.3.0",
```

Comment:
Is this downgrade of `@puppeteer/browsers` intentional? Can we remove this version change from this PR if it's not needed? There are also a lot of other version changes in this file that don't seem to be related to the focus target.